app/views/projects/_card.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="card card-sm">
+  <div class="card-body">
+    <div class="row align-items-center">
+      <div class="col-auto">
+        <span class="bg-muted text-white avatar">
+          <%= inline_svg_pack_tag(icon_path) %>
+        </span>
+      </div>
+      <div class="col">
+        <div class="font-weight-medium">
+          <%= link_to "#{resource.count if defined?(resource)} #{resource_name}", resource_path %>
```

Comment:
ez a rész nekem még mindig túlbonyolítottnak tűnik, ha valahol azt kell ellenőriznünk, hogy egy változót definiáltunk-e, akkor nem jó helyen van a kód felelőssége kezelve

ahelyett, hogy itt további kód ellenőrzéseket végeznénk, érdemes visszapasszolni az meghívónak hogy mit szeretne az elem belsejébe helyezni

```haml
<div class="font-weight-medium">
  <%= yield %>
</div>
```

majd ezután az eredeit helyen feltölteni a megfelelő tartalommal:

```haml
<%= render 'card', icon_path: '[...]/users.svg' do %>
  <%= link_to "Some link", some_path %>
<% end %>
```